### ISSUE TITLE
Setting Custom Initial Weights in a Neural Network

### ISSUE THREAD

**Issue by saltinefarmer** [lululxvi/deepxde#1771]
Hello,

I am writing a basic neural network and I want to specify all the weights that each layer starts with. These weights will naturally change as the network trains, all I care about is how they start off. Is there a way to do this?

**Comment by praksharma** [comment]
yes you can but it a really bad idea. You have to manually created a class for weight and then add it to deepxde's source code. https://stackoverflow.com/q/66724071/14598633
